src/shell/services/marketing.ts

```diff
@@ -0,0 +1,53 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+import { Announcement } from "./types";
+
+export const marketingApi = createApi({
+  reducerPath: "marketingApi",
+  baseQuery: fetchBaseQuery({ baseUrl: "https://www.zesty.io/" }),
+  endpoints: (builder) => ({
+    // Get announcements via instant api from the marketing instance
+    getAnnouncements: builder.query<Announcement[], void>({
+      query: () => "/-/instant/6-90fbdcadfc-4lc0s5.json",
```

Comment:
This needs to be a configuration value that can be different within different environments

Reply:
This ones debatable on whether it should be a config value